web/server/config/server_config.json

```diff
@@ -14,51 +14,101 @@
     "max_probe": 10
   },
   "authentication": {
-    "enabled" : false,
-    "realm_name" : "CodeChecker Privileged server",
-    "realm_error" : "Access requires valid credentials.",
-    "session_lifetime" : 300,
-    "refresh_time" : 60,
-    "logins_until_cleanup" : 30,
+    "enabled": false,
```

Comment:
Please revert the formatting of the unchanged lines.

Reply:
fixed